src/components/measureGroups/MeasureGroups.test.tsx

```diff
@@ -22,6 +18,7 @@ import { MeasureContextHolder } from "../editMeasure/MeasureContext";
 import Measure, { Group } from "../../models/Measure";
 import userEvent from "@testing-library/user-event";
 import axios from "axios";
+import { GroupScoring } from "@madie/madie-models/dist/GroupScoring";
```

Comment:
Not, much different. I think direct access like this should work work `import { GroupScoring } from "@madie/madie-models";` remember setting this up